.github/workflows/tests-karma-dashboard.yml

```diff
@@ -61,6 +71,26 @@ jobs:
       - name: Increase max number of file watchers
         run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
 
+      - name: Get Chromium executable path
+        id: chromium_path
+        run: |
+          CHROMIUM=$(node -p "const puppeteer = require('puppeteer'); puppeteer.executablePath();")
+          echo "::set-output name=chromium_path::${CHROMIUM}"
+
+      - name: Run dashboard integration tests with Percy
+        run: |
+          npx percy exec --parallel --config=percy.config.karma.yml -- \
+            npm run test:karma:dashboard -- --headless --viewport=1600:1000 --coverage --shard=$SHARD || \
+          npx percy exec --parallel --config=percy.config.karma.yml -- \
+            npm run test:karma:dashboard:retry-failed -- --headless --viewport=1600:1000 --shard=$SHARD
+        env:
+          DISABLE_ERROR_BOUNDARIES: true
+          SHARD: ${{ matrix.shard }}
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+          PERCY_BROWSER_EXECUTABLE: ${{ steps.chromium_path.outputs.chromium_path }}
+          PERCY_PARALLEL_NONCE: ${{ needs.nonce.outputs.result }}
+        if: github.event.pull_request.draft == false
+
       - name: Run integration tests
```

Comment:
This step here needs a `if: github.event.pull_request.draft == true`, at the bottom, see `tests-karma-editor.yml` for comparison.

Reply:
> This step here needs a if: github.event.pull_request.draft == true

I'm unclear here as I'm seeing  == false in `tests-karma-editor.yml`

```
if: github.event.pull_request.draft == false
```

<img width="1000" alt="Screen Shot 2022-02-23 at 5 59 36 PM" src="https://user-images.githubusercontent.com/62242/155423903-ac724e36-f630-4b24-aa4e-a4a2ed629a8a.png">

Do we want it to be `true` for the dashboard config?